packages/chart/src/components/Legend/Legend.Component.tsx

```diff
@@ -72,6 +75,12 @@ const Legend: React.FC<LegendProps> = forwardRef(({ config, colorScale, seriesHi
                   if (seriesHighlight.length > 0 && false === seriesHighlight.includes(itemName)) {
                     className.push('inactive')
                   }
+                  if (config.legend.style === 'gradient') {
```

Comment:
update TS on legend.

Reply:
Should you just return null at this point without updating the class? It looks like the next like is a return statement?

Reply:
Fixed